### ISSUE TITLE
Feature request: Line count

### ISSUE THREAD

**Issue by tonyxiao** [lukeleppan/better-word-count#26]
It's really useful especially when working with list items to get a quick count of the number of items by selecting them.

**Comment by chrisgrieser** [comment]
I was also coming here to request this. Line count is an useful alternative to sentences, for exactly the reason @tonyxiao mentioned.

**Comment by lukeleppan** [comment]
Hi thanks for the request. Sorry I'm only seeing it now. I really do like the line count idea. I am definitely adding it  to the list. Unfortunately, as I have mentioned before, updates will only start coming out again in December.

**Comment by hay** [comment]
Yeah, a line count would be great. For example, i'm now creating a list of my favourite music albums of the year, and i want to limit it to 20, so a line count would be incredibly useful. I actually thought `sentences_count` would do that, but that one works a bit more subtle.

**Comment by imeed166** [comment]
+1

**Comment by lukeleppan** [comment]
So just to update everyone. Had to rewrite the plugin. Took time off because life. Back now new update want this too probably this week.

**Comment by caioraphael1** [comment]
+1
